src/server/helpers/taxonomyLookup.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2018-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import {
```

Comment:
Since it is probably only used in forwardingApp.js i suggest moving this to that file.